fedora: Hits -Werror=format-overflow on s390x

This is "gcc (GCC) 10.0.1 20200130 (Red Hat 10.0.1-0.7)", fedora rawhide on s390x:

In function 'varnish_ask_cli',
    inlined from 'varnish_launch' at vtc_varnish.c:500:6:
vtc_varnish.c:121:3: error: '%s' directive argument is null [-Werror=format-overflow=]
  121 |   vtc_fatal(v->vl, "CLI failed (%s) = %d %u %s",
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  122 |       cmd, i, retval, r);
      |       ~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors

--- a/bin/varnishtest/vtc_varnish.c
+++ b/bin/varnishtest/vtc_varnish.c
@@ -121,7 +121,7 @@ varnish_ask_cli(const struct varnish *v, const char *cmd, char **repl)
 	i = VCLI_ReadResult(v->cli_fd, &retval, &r, vtc_maxdur);
 	if (i != 0 && !vtc_stop)
 		vtc_fatal(v->vl, "CLI failed (%s) = %d %u %s",
-		    cmd, i, retval, r);
+		    cmd != NULL ? cmd : "NULL", i, retval, r);
 	vtc_log(v->vl, 3, "CLI RX  %u", retval);
 	vtc_dump(v->vl, 4, "CLI RX", r, -1);
 	if (repl != NULL)

